learn-css-grid-by-building-a-magazine/css/style410.css

```diff
@@ -0,0 +1,117 @@
+*,
+*::before,
+*::after {
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 62.5%;
+}
+
+body {
+  font-family: 'Baskervville', serif;
+  color: linen;
+  background-color: rgb(20, 30, 40);
+}
+
+h1 {
+  font-family: 'Anton', sans-serif;
+}
+
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-family: 'Raleway', sans-serif;
+}
+
+a {
+  text-decoration: none;
+  color: linen;
+}
+
+main {
+  display: grid;
+  grid-template-columns: minmax(2rem, 1fr) minmax(min-content, 94rem) minmax(
+      2rem,
+      1fr
+    );
+  row-gap: 3rem;
+}
+
+.heading {
+  grid-column: 2 / 3;
+
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  row-gap: 1.5rem;
+}
+
+.hero {
+  grid-column: 1 / -1;
+  position: relative;
+}
+
+img {
+  width: 100%;
+  object-fit: cover;
+}
+
+.hero-title {
+  position: absolute;
+  top: 10%;
+  left: 15%;
+  color: orangered;
+  font-size: 8rem;
+}
+
+.hero-subtitle {
+  font-size: 2.4rem;
+  color: orangered;
+  text-align: center;
+}
+
+.author {
+  font-size: 2rem;
+  font-family: 'Raleway', sans-serif;
+}
+
+.author-name a:hover {
+  background-color: #306203;
+}
+
+.author-date {
+  color: rgba(255, 255, 255, 0.5);
+}
+
+.social-icons {
+  font-size: 3rem;
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+  grid-auto-flow: column;
+  grid-auto-columns: 1fr;
+  align-items: center;
+}
+
+.text {
+  grid-column: 2 / 3;
+  font-size: 1.8rem;
+  letter-spacing: 0.6px;
+  column-width: 25rem;
+  text-align: center;
+}
+
+.first-paragraph::first-letter {
+  font-size: 6rem;
+  float: left;
+  margin-right: 1rem;
+  color: orangered;
+}
+
+/* 
```

Comment:
```
/* 
Our paragraph looks really good now. The only styling we need is for the blockquote.
Create a rule for the `hr` element and set the `margin` property to `1.5rem 0`. It will add some space at the top and bottom of the blockquote. 
*/
```